src/components/common/ListSelect.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+type OwnProps = {
+  isOpen: boolean,
+  searchable?: boolean,
+  placeholder?: string,
+  items: [],
+  handleInputChange?: string => void,
+  position: Object
+};
+
+type ConnectedProps = {};
+
+type State = {};
+
+class ListSelect extends Component<OwnProps & ConnectedProps, State> {
+  render() {
+    const { top, right, left, bottom } = this.props;
+    if (!this.props.isOpen || !this.props.items) {
+      return null;
+    }
+    return (
+      <Wrapper top={top} right={right} left={left} bottom={bottom}>
+        <Title>{this.props.children}</Title>
+        {this.props.searchable && (
+          <FilterContainer>
+            <Filter
+              type="text"
+              placeholder={this.props.placeholder}
+              onChange={e => this.props.handleInputChange(e)}
+            />
+          </FilterContainer>
+        )}
+        {this.props.items.map(item => this.props.render(item))}
+      </Wrapper>
+    );
+  }
+}
+
+ListSelect.defaultProps = {
+  isOpen: false,
+  searchable: false,
+  placeholder: 'Search...',
+  top: '',
+  right: '',
+  bottom: '',
+  left: ''
+};
+
+ListSelect.propTypes = {
+  items: PropTypes.array.isRequired,
+  isOpen: PropTypes.boolean,
+  searchable: PropTypes.boolean,
+  handleInputChange: PropTypes.func,
+  top: PropTypes.string,
+  right: PropTypes.string,
+  bottom: PropTypes.string,
+  left: PropTypes.string
+};
+
+const Wrapper = styled.div`
+  z-index: 1;
+  position: absolute;
+  top: ${({ top }) => top};
```

Comment:
surround the entire css line with expression. this generates a 'top' rule no matter what